### PR TITLE
d3d12: Move d3d12 files to separate solution.

### PR DESCRIPTION
**Pull request by vlj**
It simplify solution configurations.

I'd like to move all "platform dependent" code like openAL, openGL, XAudio, PPU/LLVM, SPU/ASMJIT,  out of emucore project so that emucore only contains code that relies on a non system library.
It should then be easier to make spin off for Windows Store, Android, iOS, whatever...

### CODE REVIEW

**Comment by Nekotekina** [comment]
1) Bad copypasta of stdafx.h
2) Do not merge without DH approval

**Comment by vlj** [comment]
Fixed build and  include stdafx instead of copying it.

File: rpcs3/D3D12GSRender.vcxproj.filters [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    <ClInclude Include="Emu\RSX\D3D12\D3D12VertexProgramDecompiler.h">
+      <Filter>Fichiers d%27en-tête</Filter>
+    </ClInclude>
+    <ClInclude Include="stdafx_d3d12.h">
+      <Filter>Fichiers d%27en-tête</Filter>
+    </ClInclude>
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="Emu\RSX\D3D12\D3D12Buffer.cpp">
+      <Filter>Fichiers sources</Filter>
```
Comment (tambry) [inline rpcs3/D3D12GSRender.vcxproj.filters]: "Fichiers sources"?

Comment (Zangetsu38) [inline rpcs3/D3D12GSRender.vcxproj.filters]: @tambry French Visual ^^

**Comment by DHrpcs3** [comment]
hmm... We can move HLE modules to separate project too. But as dynamic linking library project, because modules can be LLE'd (in some far future, of course :D)

**Comment by Bigpet** [comment]
@DHrpcs3 why not just statically link it? It's not like it's going to be re-used anywhere and designing a DLL interface seems kind of like a lot of work for little to no benefit. Or is there something I'm missing.

**Comment by DHrpcs3** [comment]
@Bigpet, that allow to make clean stop/restart emulation (simply reload library). We have some problems with HLE modules reloading, as I know. 
+ we would improve linking time (that veeeery slow, atm)

**Comment by vlj** [comment]
I changed filters name.

BTW my initial intend was to write a minimal Windows Store app that can show list of games + game screen. It won't replaced the wx based gui, it's more a proof of concept. If it's possible to link and use emucore inside a Windows Store app, this should ensure there is no "hidden" dependencies on x86 (that's why I need to split asmjit and ppurecompiler) or opengl/openAL.

**Comment by vlj** [comment]
Closing, will rework it a little bit

**Pull request closed without merging**